global_body_planner/test/test_kinematics.cpp

```diff
@@ -0,0 +1,298 @@
+#include <gtest/gtest.h>
+#include <ros/ros.h>
+
+#include "global_body_planner/planner_class.h"
+#include "global_body_planner/planning_utils.h"
+#include "global_body_planner/rrt_connect.h"
+#include <quad_utils/ros_utils.h>
+
+const int N = 1000;
+const double kinematics_tol = 1e-6;
+
+TEST(GlobalBodyPlannerTest, testLeapAction) {
+
+  // Create planner and configuration
+  PlannerClass P(FORWARD);
```

Comment:
You could make a test fixture at the top that instantiates planner object and then just access it in every test.